lib/active_model_serializers/adapter/json_api/pagination_links.rb

```diff
@@ -2,14 +2,21 @@ module ActiveModelSerializers
   module Adapter
     class JsonApi < Base
       class PaginationLinks
+        MissingSerializationContextError = Class.new(KeyError)
```

Comment:
I'm happy to put a guard wherever it's to be used...  Can't think of a better spot...